vars.py

```diff
@@ -137,10 +143,22 @@ def get_force_blitz_win(self):
 
     def blitz_loss_reset(self):
         return self.blitz_loss_force_reset
-
-    def use_set_seed(self):
-        return self.set_seed
-
+    
+    def game_is_patched(self):
+        return self.patched
```

Comment:
Now is as good a time as want to mention that we don't need this type of method. We can just check `self.patched` directly

Reply:
Tried that, didn't work, didn't want it to be a blocker to progress. We can improve this later.